app/services/home.js

```diff
@@ -1,4 +1,11 @@
 'use strict';
+let errors = require('app/errors');
+let Slider = require('app/models/slider');
+let Bookshelf = require('app/bookshelf');
+
+let Sliders = Bookshelf.Collection.extend({
+    model: Slider
+});
```

Comment:
hhahah, what is happening here, i smell an antipattern, defening a collection here?
i don't have time to see why it was needed, but something is definately wrong here ..
will prolly check letta